apps/server/src/migrations/mikro-orm/Migration20241004121012.ts

```diff
@@ -0,0 +1,29 @@
+import { Migration } from '@mikro-orm/migrations-mongodb';
```

Comment:
It seems like you renamed the Migration by accident. The old migration was already in the system, so it has already been executed in some environments. I dont know what will happen when you execute this one... maybe you should test that and/or clean up

Reply:
The file name of the migration had an an additional character ("4") which messed up the order of the migrations in the IDE directory list. I tried to fix that. But yes, as the migration was already merged this wasn't a good idea. I changed it back.